workflows/metatdenovo.nf

```diff
@@ -242,10 +249,21 @@ workflow METATDENOVO {
     //
 
     ch_unpigz_contigs = UNPIGZ_MEGAHIT_CONTIGS.out.unzipped.collect { [ [ id: 'all_samples' ], it ] }
-    if( params.skip_eukulele) {} 
-    else {
-        SUB_EUKULELE(ch_unpigz_contigs)
+    
+    if( !params.skip_eukulele){
+        if( params.eukulele_db == EUKULELE_DB_PHYLODB ){
+            SUB_EUKULELE(ch_unpigz_contigs)
+        }
+        else if( params.eukulele_db == EUKULELE_DB_MMETSP){
+            SUB_EUKULELE(ch_unpigz_contigs)
         }
+        else if( params.eukulele_db == EUKULELE_DB_EUKPROT){
+            SUB_EUKULELE(ch_unpigz_contigs)
+        }
+        else if( params.eukulele_db == EUKULELE_DB_EUKZOO){
+            SUB_EUKULELE(ch_unpigz_contigs)
+        }
```

Comment:
You don't need these `if`s as they all result in the same call. (I probably confused you by referring to the ORF caller.)
What we need from these values is only the code in line 25 that means whatever the user provided as value to `eukulele_db` will be checked.